deployment/ccip/view/view.go

```diff
@@ -13,6 +13,8 @@ import (
 )
 
 type ChainView struct {
+	ChainSelector uint64 `json:"chainSelector,omitempty"`
```

Comment:
Can this be `ChainName` instead?

Reply:
The chain name is the key of the map. do you need chain name here again?

Reply:
Ah I'm confused I think, I thought the key was the chain selector

Reply:
Is addresses.json different than this ChainView?